### PR TITLE
Tweaks for archiving for enterprise demo.

### PR DESCRIPTION
**Pull request by diosmosis**
- Include exception stack trace if enabled in error message.
- If idSite does not exist, ignore it in archive invalidation.

FYI @tsteur, will be merging

### CODE REVIEW

File: core/DataAccess/Model.php [hunk 1]
```diff
@@ -203,7 +203,13 @@ public function updateArchiveAsInvalidated($archiveTable, $idSites, $allPeriodsT
...
         $dummyArchives = [];
         foreach ($idSites as $idSite) {
-            $siteCreationTime = Date::factory(Site::getCreationDateFor($idSite));
+            try {
+                $siteCreationTime = Site::getCreationDateFor($idSite);
+            } catch (\Exception $ex) {
+                continue;
+            }
+
+            $siteCreationTime = Date::factory($siteCreationTime);
```
Comment (tsteur) [inline core/DataAccess/Model.php:212]: fyi @diosmosis  in this case $siteCreationTime might be not defined or pointing to the time of a previous site?

Comment (tsteur) [reply core/DataAccess/Model.php:212]: actually forget it I see you continue above.

Comment (diosmosis) [reply core/DataAccess/Model.php:212]: It should skip to the next iteration if it can't be set?

Comment (tsteur) [reply core/DataAccess/Model.php:212]: Totally. sorry about that

Comment (diosmosis) [reply core/DataAccess/Model.php:212]: oh, github showed me the first comment, but not the second...

**Pull request merged**